### PR TITLE
fix: use `BIC_OVERWRITE` when `cp`ing static/ too

### LINKED ISSUE #19: Hook `BIC_OVERWRITE` into copying static files
`BIC_OVERWRITE` sets the bash `noclobber` which is good for file redirection, but I think I need to add a bit of logic to _also_ use it in the `cp` command for copying `static/` -> `/`.

### PR DESCRIPTION
**Pull request by Pinjasaur**
closes #19 

Initially tried to use `cp` with `--no-clobber/-n` to fail if trying to copy a file over that already exists. But, depending on the OS the behavior isn't consistent, particularly Ubuntu which is what I use for the Dockerfile base along with the CI runner.

I pivoted to using `cp` `--interactive`-ly and `echo`ing in `n` so it _would_ fail.

Also: bumped a few different versions of things.

### CODE REVIEW

File: bic [hunk 1]
```diff
@@ -366,7 +366,12 @@ build_feed() {
 build_static() {
   [[ ! -d "${SRC_DIR}"/static ]] && return
   log "Building /static -> /"
-  cp -a "${SRC_DIR}"/static/. "${DEST_DIR}"
+  # Archive by default
+  local args=(-a)
+  # And run interactively so as to not clobber files unless explicitly disabled
+  [[ -z "${BIC_OVERWRITE:-}" ]] && args+=(-i)
+  # shellcheck disable=SC2216
+  echo n | cp "${args[@]}" "${SRC_DIR}"/static/. "${DEST_DIR}"
```
Comment (Pinjasaur) [inline bic:374]: don't love this, but wasn't able to immediately come up with another way to accomplish it https://www.shellcheck.net/wiki/SC2216

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,15 +15,15 @@ on:
 
 env:
   IMAGE_NAME: bic
-  PANDOC_VERSION: 2.17.0.1
-  BATS_VERSION: 1.5.0
+  PANDOC_VERSION: 3.2.1
+  BATS_VERSION: 1.11.0
```
Comment (Pinjasaur) [inline .github/workflows/ci.yml:19]: it's been a couple years...

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -15,15 +15,15 @@ on:
...
 
 jobs:
   # Run tests.
   # See also https://docs.docker.com/docker-hub/builds/automated-testing/
   test:
     # in sync with Dockerfile
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```
Comment (Pinjasaur) [inline .github/workflows/ci.yml:26]: forgot to bump this when I bumped the Dockerfile 🙃

**Comment by Pinjasaur** [comment]
@mfossen if you're up for a review I'd appreciate the 👀

**Review by mfossen — APPROVED** [review]
LGTM

**Pull request merged**